repo/packages/C/confluent-connect/2/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "image": "dbtucker/cp-kafka-connect"
```

Comment:
Did you want an explicit tag on this image?  This is going to default to `latest` I think.